src/sentry/incidents/tasks.py

```diff
@@ -182,7 +182,7 @@ def handle_trigger_action(
     action_id: int,
     incident_id: int,
     project_id: int,
-    method: str,
+    method: Literal["fire", "resolve"],
```

Comment:
I couldn't find a better way to do this, my goal was to make it obvious what the values are but since I can't pass the enum through an async task for pickle reasons this was the best I could come up with.

Reply:
Idk if it helps, but i was able to get an enum working the way you're trying to like this: https://github.com/getsentry/sentry/blob/master/src/sentry/incidents/models/alert_rule.py#L50